types/knockout.mapping/tslint.json

```diff
@@ -50,21 +46,17 @@
         "no-unnecessary-qualifier": false,
         "no-unnecessary-type-assertion": false,
         "no-useless-files": false,
-        "no-var-keyword": false,
         "no-var-requires": false,
         "no-void-expression": false,
         "no-trailing-whitespace": false,
         "object-literal-key-quotes": false,
-        "object-literal-shorthand": false,
         "one-line": false,
         "one-variable-per-declaration": false,
-        "only-arrow-functions": false,
-        "prefer-conditional-expression": false,
-        "prefer-const": false,
         "prefer-declare-function": false,
         "prefer-for-of": false,
         "prefer-method-signature": false,
         "prefer-template": false,
+        "quotemark": [true, "double", "avoid-escape" ],
```

Comment:
The Prettier configuration for DT uses single quotes. Is this lint rule necessary?

Reply:
Explained in other comment.